src/Message.h

```diff
@@ -54,6 +54,19 @@ class MessageHandler {
 #undef MUMBLE_MH_MSG
 };
 
+/// UDPMessageTypeIsValidVoicePacket checks whether the given
+/// UDPMessageType is a valid voice packet.
+inline bool UDPMessageTypeIsValidVoicePacket(MessageHandler::UDPMessageType umt) {
+	switch (umt) {
```

Comment:
23:28:30 <Whooa21> mkrautz: ../../src/Message.h:60:9: warning: enumeration value ‘UDPPing’ not handled in switch [-Wswitch]